packages/core/src/theme-default/components/Overview/index.tsx

```diff
@@ -26,6 +26,7 @@ export function Overview() {
     siteData,
     page: { routePath },
   } = usePageData();
+  const cleanUrls = !!usePageData()?.siteData?.route?.cleanUrls;
```

Comment:
`siteData` has beed destructed before so there is no need to call `usePageData`. 
```ts
const cleanUrls = siteData.route?.cleanUrls
```